src/hono-base.ts

```diff
@@ -23,6 +23,8 @@ import type {
 } from './types'
 import { getPath, getPathNoStrict, getQueryStrings, mergePath } from './utils/url'
 
+export const composedHandler = Symbol('composedHandler')
```

Comment:
Since `composedHandler` is a constant like `METHODS`, maybe `COMPOSED_HANDLER` would be better. What do you think?

```ts
export const COMPOSED_HANDLER = Symbol('composedHandler')
```